docs/config.yaml

```diff
@@ -0,0 +1,55 @@
+baseURL: https://rgeyer.github.io/hugoactionstest/
```

Comment:
After this PR is merged. Follow [these directions](https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-first-deployment-with-github_token) to point gh-pages to the newly created branch.

Then this line can be changed to `https://grafana.github.io/grizzly/` and pushed, which will trigger the final github action execution and *actual* deployment of the docs.